_blog/2018/soft-multitenancy.md

```diff
@@ -290,15 +276,37 @@ technology, ex. Kubernetes, rather than improvements in Istio capabilities.
 ## Future work
 Allowing a single Istio control plane to control multiple meshes would be an obvious next
 feature. An additional improvement is to provide a single mesh that can host different
-tenants with some level of isolation and security between the tenants.
-A [document](https://docs.google.com/document/d/14Hb07gSrfVt5KX9qNi7FzzGwB_6WBpAnDpPG6QEEd9Q)
+tenants with some level of isolation and security between the tenants.  This could be done
+by partitioning within a single control plane using the same logical notion of namespace as 
+Kubernetes. A [document](https://docs.google.com/document/d/14Hb07gSrfVt5KX9qNi7FzzGwB_6WBpAnDpPG6QEEd9Q)
 has been started within the Istio community to define additional use cases and the
 Istio functionality required to support those use cases.
 
-## Issues
+## Issues, limitiations or caveats
+### Known issues 
 * The CA (Certificate Authority) and mixer Istio pod logs from one tenant's Istio control
 plane (ex. *istio-system* `namespace`) contained 'info' messages from a second tenant's
 Istio control plane (ex *istio-system1* `namespace`).
+### Challenges with the different models
+At the beginning of this blog a few different models of multitenancy were described.  This section will
+briefly discuss some of challenges with those models with the current state of Istio or Kubernetes 
+code. 
+
+Current Istio capabilities are poorly suited to support the first model as it lacks
+sufficient RBAC capabilities to support cluster versus tenant operations. Additionally,
+having multiple tenants under one mesh is too insecure with the current mesh model and the 
+way Istio drives configuration to the envoy proxies. 
+
+Regarding the 2nd option the current Istio paradigm assumes a single mesh per Istio control
+plane. The needed changes to support this model are substantial. They would require 
+finer grained scoping of resources and security domains based on namespaces, as well as,
+additional Istio RBAC changes. This model will likely be addressed by future work, but not
+currently possible. 
+
+The fourth model doesn’t satisfy most use cases, as most cluster administrators prefer
+a common Kubernetes control plane which they provide as a PaaS to their tenants.
+Additionally, case 4 is easily provided in many environments already so is not interesting to 
+further describe here.
```

Comment:
How is , or what is an example of, case 4 'easily provided in many environments already'? Now that this section is moved down to the end of the blog, not sure this last sentence is adding much at all. Perhaps remove.